### PR TITLE
[2 of 8: Service Updates & Upgrades] Introduce SegmentedProgressBar

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

**Comment by kennyt** [comment]
merging on test pass

**Comment by daiyiwhale** [comment]
Looks great!

File: src/js/components/charts/SegmentedProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    return (
+      <StackedProgressBar className={this.props.stackedProgressBarClassName}
+        progressState={segment.upgradeState} key={index}
+        secondaryLabel={segment.label} />
+    );
+  }
+
+  render() {
+    let props = this.props;
+    let classes = classNames(props.className);
```
Comment (daiyiwhale) [inline src/js/components/charts/SegmentedProgressBar.js]: Oh oops, I noticed this afterwards, but I don't believe we need `classNames` here (or the `let classes` at all probably)

Comment (jfurrow) [inline src/js/components/charts/SegmentedProgressBar.js]: Removed here #203

**Pull request merged**